src/SwiftUI/SwiftUI/ModifiedView.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+    [SwiftImport (SwiftUILib.Path)]
+    public record ModifiedView<TView, TViewModifier> : View
```

Comment:
I believe this type is called `ModifiedContent` ... https://developer.apple.com/documentation/swiftui/modifiedcontent/